hw/dv/tools/dvsim/common_sim_cfg.hjson

```diff
@@ -137,10 +137,10 @@
   // Project defaults for VCS
   vcs_cov_cfg_file: "{{build_mode}_vcs_cov_cfg_file}"
   vcs_unr_cfg_file: "{dv_root}/tools/vcs/unr.cfg"
-  vcs_cov_excl_files: ["{dv_root}/tools/vcs/common_cov_excl.el"]
+  vcs_cov_excl_files: []
 
   // Build-specific coverage cfg files for VCS.
-  default_vcs_cov_cfg_file: "-cm_hier {dv_root}/tools/vcs/cover.cfg"
+  default_vcs_cov_cfg_file: "-cm_hier {dv_root}/tools/vcs/cover.cfg+{dv_root}/tools/vcs/common_cov_excl.cfg"
   cover_reg_top_vcs_cov_cfg_file: "-cm_hier {dv_root}/tools/vcs/cover_reg_top.cfg"
```

Comment:
We could have a problem here. The `default` build adds the exclusions on the TL signals below, but the `cover_reg_top` (and possibly other) builds do not, so in the merged coverage, the signals below will still show up as uncovered.

Reply:
It's actually fine. As long as we apply the exclusion on the base/default build, the other build can't enable those exclusion again.

I did a few experiments.
1. only add this common_cov_excl.cfg to default build, it works as expected.
2. only add this common_cov_excl.cfg to cover_reg_top build, it doesn't work and still shows those signals uncovered
3. enable toggle coverage in sub modules (rather than just the dut) in build cover_reg_top, only dut level toggle coverage is enabled.
4. when merge coverage, put cover_reg_top vdb as the base coverage (put it before default.vdb), only reg_top is enabled, all other module's coverage is excluded.

So, I think we only need to add exclusion for the default build. And make sure we always uses default coverage vdb as base (so far it is the case, but not sure if this can be always guaranteed).

Reply:
It doesn't seem right to leave this to chance. We process our HJsons in Python which does not always guarantee lists are concatenated in the right order all the time.

Reply:
Ohh, this is a string and not a list, so what I said does not apply. 

Though, however we need to convert this HJSon variable to a list instead to add support for more user-defined `cm_hier` configs. Lets move ahead with this but discuss more offline.

Reply:
yeah, we need to make sure the vdb order is right for coverage merge. it should be `urg -dir default.vdb -dir cover_reg_top.vdb` (default.vdb is always the first one). Looks like we need to fix it in dvsim.

Reply:
oh, what I said is a different thing: ensure the vdb order is right.